src/Groups/perm.jl

```diff
@@ -323,24 +323,51 @@ true
 ```
 """
 function perm(g::PermGroup, L::AbstractVector{<:IntegerUnion})
-   x = GAPWrap.PermList(GapObj(L;recursive=true))
+   x = _make_gap_perm(degree(g), L)
    @req x !== GAP.Globals.fail "the list does not describe a permutation"
-   @req (length(L) <= degree(g) && x in GapObj(g)) "the element does not embed in the group"
+   @req x in GapObj(g) "the element does not embed in the group"
    return PermGroupElem(g, x)
 end
 
 perm(g::PermGroup, L::AbstractVector{<:ZZRingElem}) = perm(g, [Int(y) for y in L])
 
 function (g::PermGroup)(L::AbstractVector{<:IntegerUnion})
-   x = GAPWrap.PermList(GapObj(L;recursive=true))
-   @req (length(L) <= degree(g) && x in GapObj(g)) "the element does not embed in the group"
+   x = _make_gap_perm(degree(g), L)
+   @req x in GapObj(g) "the element does not embed in the group"
    return PermGroupElem(g, x)
 end
 
+function _make_gap_perm(deg::Int, L::Vector{<:IntegerUnion})
+  @req 0 <= deg "degree is $deg, must be non-negative"
+  @req deg <= 2^32 "degree is $deg, must be less than 2^32"
```

Comment:
```suggestion
  @req deg < 2^32 "degree is $deg, must be less than 2^32"
```
To match the error message